qualtran/bloqs/hamiltonian_simulation/hamiltonian_simulation_by_gqsp.ipynb

```diff
@@ -80,7 +80,13 @@
     "This therefore block-encodes $e^{-iHt}$ in the block where the signal qubit and selection registers are all $|0\\rangle$.\n",
     "\n",
     "#### References\n",
-    " - [Generalized Quantum Signal Processing](https://arxiv.org/abs/2308.01501).     Motlagh and Wiebe. (2023). Theorem 7, Corollary 8.\n"
+    " - [Generalized Quantum Signal Processing](https://arxiv.org/abs/2308.01501).     Motlagh and Wiebe. (2023). Theorem 7, Corollary 8. \n",
```

Comment:
In the docstring, we're now (lightly) enforcing a particular format for our references. Can you remove the additional indentation from the second line of the reference please? (if there is more than one reference: separate them by a blank line)

Reply:
oops forgot to fix in the last PR, fixed now.